msal/token_cache.py

```diff
@@ -38,13 +38,19 @@ def __init__(self):
     def find(self, credential_type, target=None, query=None):
         target = target or []
         assert isinstance(target, list), "Invalid parameter type"
+        target_set = set(target)
```

Comment:
is target scopes? if yes, do you have a place where you remove dupes?

Reply:
Yes `target` is exactly `scope` in Unified Schema's dialect.

The `set(...)` method creates a native set object in Python. It automatically deduplicates the scopes.